### DISCUSSION TITLE
Player triggered creature attack movement / animation is not replicated to other players

### DISCUSSION THREAD

**Discussion by TheNinjaScout** [FakeFishGames/Barotrauma#14674] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Triggering a creature attack as a player controlled monster or human with genetic appendages that can use creature attacks shows your own character moving to attack, but other players do not see you move.
Example: https://youtu.be/Str6ZdJZ9gc

### Reproduction steps

1. Join a multiplayer server
2. Have someone else control a character that has access to a creature attack
3. Watch them attack stuff and note how the only movement you see is their main collider moving on certain attacks without any actual movement of the limbs

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next update. Closing discussion.
